Stack/Opc.Ua.Core/Security/Certificates/CertificateValidator.cs

```diff
@@ -22,7 +22,7 @@ namespace Opc.Ua
     /// <summary>
     /// Validates certificates.
     /// </summary>
-    public class CertificateValidator
+    public class CertificateValidator : X509CertificateValidator
```

Comment:
This is wired because the generic class name inherits from the specific classname.

Reply:
to get rid of WcfValidator

---
In reply to: [155840316](https://github.com/OPCFoundation/UA-.NETStandard/pull/284#discussion_r155840316) [](ancestors = 155840316)